integration_tests/tests/apply/about_the_person/equality_and_diversity/complete_disability_page.cy.ts

```diff
@@ -58,10 +58,7 @@ context('Visit "About the person" section', () => {
 
     page.clickSubmit()
 
-    // I return to the task list page
-    const taskListPage = Page.verifyOnPage(TaskListPage)
-
-    // I see that the task has been completed
-    taskListPage.shouldShowTaskStatus('equality-and-diversity-monitoring', 'Completed')
+    // I am taken to the 'sex and gender' page
+    Page.verifyOnPage(SexAndGenderPage)
```

Comment:
I think we need to have the `pages/apply/sexAndGenderPage` added also in this commit if this is going to pass?